### DISCUSSION TITLE
Does Browser Detection library supports PHP 8.2?

### DISCUSSION THREAD

**Discussion by tejswini1991** [community/community#76292] [Programming Help]
### Select Topic Area

Question

### Body

Hi,
I am using below PHP library by Wolfcast to detect browser . Since my client going to shift the project to PHP 8.2 so does browser detection library support php 8.2?
https://github.com/Wolfcast/BrowserDetection

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @tejswini1991 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
